src/transformStateWithClones.js

```diff
@@ -8,6 +8,39 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const statesArray = [];
+  const cloneState = { ...state };
```

Comment:
https://github.com/mate-academy/js_stateful-clones/pull/2699#discussion_r1306618210
fix this const naming

Reply:
done